### PR TITLE
Added edam annotations to egsea

### PR DESCRIPTION
**Pull request by LucieLamothe**
topics: Gene expression, Systems biology
operations: Gene-set enrichment analysis

### CODE REVIEW

File: tools/egsea/egsea.xml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 <tool id="egsea" name="EGSEA" version="1.20.0">
     <description> easy and efficient ensemble gene set testing</description>
+    <edam_topics>
```
Comment (bgruening) [inline tools/egsea/egsea.xml:3]: @LucieLamothe if we have a bio.tools ID we do not need the EDAM terms, as they will be retrieved automatically from bio.tools.

Comment (LucieLamothe) [reply tools/egsea/egsea.xml:3]: Oh I didn't know that, it's great! :) Shall I add bio.tools id instead when they exists then?

Comment (nsoranzo) [reply tools/egsea/egsea.xml:3]: Yes, that's the idea. Thanks!

**Comment by bernt-matthias** [comment]
Then I guess I can close this one

**Pull request closed without merging**